package.json

```diff
@@ -57,7 +57,8 @@
     "decanter": "^7.2.0",
     "eslint": "^8.50.0",
     "eslint-config-next": "^13.4.13",
-    "netlify-plugin-vault-variables": "^2.0.0",
+    "netlify-cli": "^17.22.1",
+    "netlify-plugin-vault-variables": "github:SU-SWS/netlify-plugin-vault-variables#nojira/nextjs-plugin-5",
```

Comment:
There was an incompatibility with @netlify/plugin-nextjs version 5.2 that this branch fixes. Let's wait until that PR gets merged and a new release gets punched before merging this one.